content/en/docs/reference/node/seccomp.md

```diff
@@ -0,0 +1,117 @@
+---
+content_type: reference
+title: Seccomp and Kubernetes
+weight: 80
+---
+
+<!-- overview -->
+
+Seccomp stands for secure computing mode and has been a feature of the Linux
+kernel since version 2.6.12. It can be used to sandbox the privileges of a
+process, restricting the calls it is able to make from userspace into the
+kernel. Kubernetes lets you automatically apply seccomp profiles loaded onto a
+{{< glossary_tooltip text="node" term_id="node" >}} to your Pods and containers.
+
+## Seccomp fields
+
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+
+There are three ways to specify a seccomp profile for a
+{{< glossary_tooltip text="pod" term_id="pod" >}}:
+
+- for the whole Pod using [`spec.securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context)
+- for a single container using [`spec.containers[*].securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1)
+- for an [ephermal container](/docs/concepts/workloads/pods/ephemeral-containers) using [`spec.ephemeralContainers[*].securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-2)
+
+{{% code_sample file="pods/security/seccomp/fields.yaml" %}}
+
+The Pod in the example above runs as `Unconfined`, while the
+`ephemeral-container` specifically defines `RuntimeDefault`. If the ephemeral
+container would not have set the `securityContext.seccompProfile` field
+explicitly, then the value would be inherited by the Pod. The same applies to
+the container, which runs a `Localhost` profile `my-profile.json`.
+
+Generally speaking, fields from (ephemeral) containers have a higher priority
+than the Pod level value, while containers which do not set the seccomp field
+are being inherited by the Pod.
+
+{{< note >}}
+It is not possible to apply a seccomp profile to a Pod or container running with
+`privileged: true` set in the container's `securityContext`. Privileged
+containers always run as `Unconfined`.
+{{< /note >}}
+
+The following values are possible for the `seccompProfile.type`:
+
+`Unconfined`
+: The workload runs without any seccomp restrictions.
+
+`RuntimeDefault`
```

Comment:
We need to speak of a default value here. Do we need the same callout as https://kubernetes.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads

Reply:
Added more context and another paragraph about the details of that profile.